guides/source/active_storage_overview.md

```diff
@@ -490,7 +490,25 @@ directly from the client to the cloud.
     ```erb
     <%= form.file_field :attachments, multiple: true, direct_upload: true %>
     ```
-3. That's it! Uploads begin upon form submission.
+3. Enable Cross-Origin Resource Sharing (CORS): You need to allow direct upload from browser to your cloud storage by enabling CORS. Without this, It will not pass the browser's preflight check. you will see HTTP 403 error when trying to upload. Below are instructions for Amazon S3. if you do not use S3, then please look at the provider specified documentation on how to enable CORS.
+
+    Amazon S3: Goto AWS Management Console -> S3 Bucket -> Permissions -> CORS Configuration and add CORS policy. A sample policy for non-production use is given below.
+
+  ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+    <CORSRule>
+        <AllowedOrigin>http://localhost:3000</AllowedOrigin>
+        <AllowedMethod>PUT</AllowedMethod>
+        <MaxAgeSeconds>3000</MaxAgeSeconds>
+        <AllowedHeader>Content-Type</AllowedHeader>
+        <AllowedHeader>Content-MD5</AllowedHeader>
+    </CORSRule>
+    </CORSConfiguration>
+  ```
+  For more details, please see [Amazon CORS Documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html)
+
```

Comment:
I don't know if it makes sense to put a specific providers directions in the docs. @georgeclaghorn what are your thoughts on this?

Reply:
I'd suggest the AllowedOrigin should be wildcard because domain name had to be "*" to work on our app provisioned using terraform